crates/nostr/src/event/kind.rs

```diff
@@ -84,6 +84,38 @@ pub enum Kind {
     MuteList,
     /// Pin List (NIP51)
     PinList,
+    /// Bookmarks (NIP51)
```

Comment:
Let's link nips when they are mentioned!
```suggestion
    /// Bookmarks [NIP51](https://github.com/nostr-protocol/nips/blob/master/51.md)
```